### DISCUSSION TITLE
How to encrypt your codes

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#146] [Q&A] [answered]
How to encrypt your codes. I wanted to encrypt my js files. Is there any official software ?. 

I used online encryption tools it worked but it slows the program and it's not too good. I wanted to convert those js files to bin files.

**Comment by zdunex25** [comment, marked as answer]
You do not simply convert JS to BIN, build with Zeus CLI and you will get BIN files.
